tests/test_object_detection.py

```diff
@@ -592,6 +596,38 @@ def test_find_label_issues_per_box():
     assert (issues_per_box[2] == np.array([False, True, False, False])).all()
 
 
+def test_get_object_count():
+    label_count, pred_count = get_object_count(labels, predictions)
+    assert (label_count == np.array([len(sample["bboxes"]) for sample in labels])).all()
+    assert (pred_count == np.array([sum([len(cl) for cl in pred]) for pred in predictions])).all()
+
+
+def test_get_bbox_sizes():
+    label_boxes, pred_boxes = get_bbox_sizes(labels, predictions)
+    assert np.all(label_boxes >= 0)
+    assert np.all(pred_boxes >= 0)
+
+
+def test_get_class_distribution():
+    lab_count, pred_count = collections.defaultdict(int), collections.defaultdict(int)
+
+    for sample in labels:
+        for cl in sample["labels"]:
+            lab_count[cl] += 1
+
+    for sample in predictions:
+        for i, cl in enumerate(sample):
+            if len(cl) > 0:
+                pred_count[i] += len(cl)
+
+    lab_total, pred_total = sum(lab_count.values()), sum(pred_count.values())
+    lab_freq_ans = {k: round(v / lab_total, 2) for k, v in lab_count.items()}
+    pred_freq_ans = {k: round(v / pred_total, 2) for k, v in pred_count.items()}
+    lab_freq, pred_freq = get_class_distribution(labels, predictions)
+    assert lab_freq == lab_freq_ans
+    assert pred_freq == pred_freq_ans
+
```

Comment:
I'm delighted to see your contributions – they're truly fantastic! Your dedication to the open-source community is greatly valued.Your contributions mean a lot to us!

Reply:
Thanks a lot for the encouragement @aditya1503!